### ISSUE TITLE
Update general-info/accessibility

### ISSUE THREAD

**Issue by andyhickner** [code4lib/2021.code4lib.org#9]
- [x] Re-write or delete microphone sections

**Comment by ranti** [comment]
Do we know who handles accessibilty email address? Acccessibility Committee on 2021 Committee wiki page does not have anybody who lead the team.

**Comment by dzoladz** [comment]
According to the overall project's Trello, the formation of the accessibility committee has a due date set at the beginning of 2021... so I guess it's not too surprising that no one has signed on to lead that team yet.

In `_data/conf.yml`, there's a general gmail account that's listed:
`accessibility-email: 'code4lib.accessibility@gmail.com'`

I'll find out who manages access to that account.

**Comment by dzoladz** [comment]
FYI - the accessibility committee has been formed: https://wiki.code4lib.org/Code4Lib_2021_Conference_Committees#Accessibility_Committee

**Comment by ranti** [comment]
Contacted acesssibility committee to help with general-info/accessibilty content.

**Comment by ranti** [comment]
**Adding issue per #55** 

**Accessibility page**

- [x] slide submission email address is an empty link in action 2 - send to conference planning email address

- [x] links under sharing slides should be part of the text, not written out urls (awful to hear on a screen reader)
![Screen Shot 2020-12-07 at 3 43 36 PM](https://user-images.githubusercontent.com/10561752/101403286-033baf80-38a3-11eb-83b1-d059a684c29e.png)

**Comment by dzoladz** [comment]
We should have some updates for this content from the Accessibility Committee in the next few days

**Comment by ranti** [comment]
PR 104 for review only. A lot of new/changed info.

Addressed:
- Hide the info about microphone and podium
- Use conditional logic to show/hide the slide submission repository
- URLs are now embedded as part of the text

**Comment by ranti** [comment]
Still need: 
- [ ] email address for slide submission 
- [ ] correct link for the slide repository
